### DISCUSSION TITLE
Favorite lists in GitHub Mobile are not accessible from GitHub.com.

### DISCUSSION THREAD

**Discussion by Saksham093** [community/community#122398] [Mobile]
How to access Favorite lists saved  GitHub Mobile from GitHub.com.

The Favourites in Mobile is much better than list.

![Screenshot_20240505-173258.jpg](https://github.com/community/community/assets/69385787/1da5df8c-a621-48e3-999a-6a75f8633561)

**Comment by TankEngine-ish** [comment]
As far as I know this option is not available on github desktop or via the website. Only stars and lists.

**Reply by Saksham093** [reply]
Let's wait to add this feature.

**Comment by salma-elbakkouri** [comment]
You can only access starred repositories on Github website

**Reply by Saksham093** [reply]
Yeah, But this feature is cool.
